### DISCUSSION TITLE
Custom color map in WFS layers

### DISCUSSION THREAD

**Discussion by samareshdasglobal** [TerriaJS/terriajs#7183] [Q&A] [answered]
Hello,

I have published two WFS layers in GeoServer created from shapefiles. Both layers are loading fine in TerriaMap with the default colormap. However, the same colormap is showing for seven values in both cases. One layer represents climb values and the other represents nature values. I need to set different colormaps for each layer. How can I customize these colormaps either through configuration or by changing the code?

Thanks
Samaresh

**Comment by nf-s** [comment, marked as answer]
Hi @samareshdasglobal 

We have a quick guide on using the "Edit Style" feature to do this here

- https://github.com/TerriaJS/terriajs/discussions/6422#discussioncomment-3176804

Let me know if need any more info

Thanks!
